4-forms-validations/form.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Html Form</title>
+
+    <style>
+      body {
+        display: flex;
+        flex-direction: column;j
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+      }
+      form {
+        display: flex;
+        flex-direction: column;
+      }
+      label {
+        margin: 7px;
+      }
+      .hobbies {
+        display: flex;
+        flex-direction: column;
+      }
+      fieldset {
+        margin-block: 5px;
+      }
+    </style>
+  </head>
+  <body>
+    <header><h1>Simple Form</h1></header>
+    <main>
+      <form>
+        <label for="name"
+          >First Name <br />
+          <input type="text" name="first-name" id="fn" required />
+        </label>
```

Comment:
Please check the `id` and `for` attributes for the `input` and the `label` tags respectively. Take a look to the Developer Mozilla description about them. 

![Screen Shot 2021-11-28 at 5 42 51 PM](https://user-images.githubusercontent.com/36536646/143789228-ac2d5fee-e2c6-45d5-9fb2-bf9e14d2534d.png)

Reply:
Yes, I was confused with this attribute, thanks